### PR TITLE
Added module for User Account

### PR DESCRIPTION
**Pull request by chaitanyaavi**
##### SUMMARY

Added module for User Account

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

network/avi/vi_useraccount.py
##### ANSIBLE VERSION

```
ansible 2.6.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/vinay/git/avitest/local/lib/python2.7/site-packages/ansible
  executable location = /home/vinay/git/avitest/bin/ansible
  python version = 2.7.6 (default, Nov 23 2017, 15:49:48) [GCC 4.8.4]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by grastogi23** [comment]
@chaitanyaavi  pl. update the component name and re-test with the ansible devel.

File: lib/ansible/modules/network/avi/avi_useraccount.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+author: Chaitanya Deshpande (chaitanya.deshpande@avinetworks.com)
+
+short_description: Avi UserAccount Module
+description:
+    - This module can be used for updating the password of a user.
+    - This module is useful for setting up admin password for Controller bootstrap.
+version_added: 2.6
+requirements: [ avisdk ]
+options:
+    old_password:
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_useraccount.py:43]: This isn't clear to me.

Comment (chaitanyaavi) [reply lib/ansible/modules/network/avi/avi_useraccount.py:43]: username and password are part of common argument spec so to update a password user has to provide new and old both passwords. Using the password field in common argument spec user will specify the new password and with the option old_password user will specify old password or default password (Avi controller image is shipped with a default password can be used once only to update the password to bootstrap)

**Review by grastogi23 — APPROVED** [review]

**Comment by grastogi23** [comment]
Since, this is user password change we do need both password. Also, the module is created such that it can be idempotent and it allows user to not pass old_password all the time.

**Comment by grastogi23** [comment]
+shipit

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged into `devel` will be released in Ansible 2.6.0

**Pull request merged**